### DISCUSSION TITLE
Is terraformer capable of generating tf 1.0.x code?

### DISCUSSION THREAD

**Discussion by siepa3** [GoogleCloudPlatform/terraformer#1154] [General]
The readme states tf 0.13 compatibility.

**Comment by lorayin** [comment]
my local terraform is v1.0. Also get similar problem when try to terraform init in terraformer generated folder, show error like:
" Initializing the backend..
╷
│ Error: Invalid legacy provider address
│
│ This configuration or its associated state refers to the unqualified provider "aws".
│
│ You must complete the Terraform 0.13 upgrade process before upgrading to later versions.

**Comment by healthyGenZ** [comment]
I am having same issue with Datadog , Terraform says to run 0.13 upgrade but then my terraform is 1.x and 0.13 upgrade is available only in that specific release. Is only way to install old version of terraform ? Any other ways using which we can have configuration changes pushed to Datadog via .tf files ?

**Reply by jittoshi** [reply]
I am running into this same issue as well. Any help on this would be great. I am trying to run `terraform init` and I am receiving this error: 
```
This configuration or its associated state refers to the unqualified provider "aws".
│ You must complete the Terraform 0.13 upgrade process before upgrading to later versions.
```
I try running `terraform 0.13upgrade` and I get 

`Terraform has no command named "0.13upgrade".`
